sites/example-project/src/pages/test-analysis.md

```diff
@@ -11,7 +11,7 @@
         {x: 17, y:18, y2: 26, y3: 16}
     ]
 
```

Comment:
If users insert script tags and declare their own vars, we could get var name collisions as of today (there is a query by the same name on the page).  However, the error message is pretty clear in such cases.

Reply:
Agreed. The user bringing their own data here is pretty advanced as well.